src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/display/ChunkOptionsPopupPanel.java

```diff
@@ -202,36 +203,38 @@ else if (value == OUTPUT_SKIP_THIS_CHUNK)
       });
       
       int row = includeChunkNameUI ? 1 : 0;
-      FormLabel outputLabel = new FormLabel("Output:");
+      FormLabel outputLabel = new FormLabel(constants_.outputColon());
       nameAndOutputGrid.setWidget(row, 0, outputLabel);
       nameAndOutputGrid.setWidget(row, 1, outputComboBox_);
       outputLabel.setFor(outputComboBox_);
       
       panel_.add(nameAndOutputGrid);
       
       showWarningsInOutputCb_ = makeTriStateToggle(
-            "Show warnings",
+            constants_.showWarnings(),
             "warning");
       panel_.add(showWarningsInOutputCb_);
-      
+
+      // TODO these 'option's seem to be used later(marked with TODO), do these need translations?
```

Comment:
Yes, these are used as you described, and should not be localized. Please remove the TODO comments.